client/HomePage.js

```diff
@@ -5,10 +5,10 @@ import {Link} from 'react-router-dom'
 export default class HomePage extends Component {
   render() {
     return (
-      <div>
+      <div className="items">
         <main>
```

Comment:
you can add the class "mainPage" to this tag, and then reference tags within it (like h1 and h2) like so.

`mainPage h1 {
  font-size: 2rem;
 //etc..
`
so you don't have to add the mainPage class to every element within it. Just add it to the closest common parent.